src/main/java/org/example/JsonValidator.java

```diff
@@ -0,0 +1,38 @@
+package org.example;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONException;
+
+public class JsonValidator {
+
+    private final URLValidator urlValidator;
+    private final PhoneNumberValidator phoneNumberValidator;
+
+    public JsonValidator() {
+        this.urlValidator = new URLValidator();
+        this.phoneNumberValidator = new PhoneNumberValidator();
```

Comment:
These validators must be passed as a Map<String, Validator> parameters, similar to what we did in CSV validator